crates/compiler/src/util.rs

```diff
@@ -11,6 +11,7 @@ use std::{
 };
 
 mod v_str;
+pub mod ast_print;
```

Comment:
YAML snapshot has already been implemented in the repo. Rolling out another ast print doesn't deserve much.

Reply:
I think it is not conflict, yaml used to snapshot testing,  this util give developer a quick visualize for ast.
I would like to add a new feature `debug` , this mod only compile when `debug` feature is enable.
What do you think?

Reply:
I'm afraid the debug doesn't pay off. 
First, the CLI is just a proof concept of Vue compiler in Rust. It isn't the main goal of this project. Writing code specifically for it won't get used much.
Second, current architecture has richer data structures since it has multiple passes in compiling. A debugging ast print would need to cover all passes like token, parsing, IR and final JS output. That will probably be a lot of code that requires mechanic labor to write.
Finally, adding a debug feature gate is, well, one more gate. It needs some maintenance as well.

Reply:
Would it be better move this to cli crate? Because currently there is nothing to output due to the incomplete transform and codegen